maven-plugin/pom.xml

```diff
@@ -20,12 +20,12 @@
   <modelVersion>4.0.0</modelVersion>
 
   <parent>
-    <groupId>org.projectnessie</groupId>
-    <artifactId>nessie-apprunner-parent</artifactId>
+    <groupId>org.projectnessie.nessie-runner</groupId>
+    <artifactId>nessie-runner-parent</artifactId>
     <version>0.29.1-SNAPSHOT</version>
   </parent>
 
-  <artifactId>nessie-apprunner-maven-plugin</artifactId>
+  <artifactId>nessie-runner-maven-plugin</artifactId>
   <packaging>maven-plugin</packaging>
 
   <name>Nessie Quarkus Application Runner Maven Plugin</name>
```

Comment:
elsewhere (i.e. common module) we are dropping `Application` from `Quarkus Application` (maybe another term we should grep for)

Reply:
updated